docs/feature-gates.md

```diff
@@ -32,30 +32,31 @@ edit the Agent configuration in the
+| `NetworkPolicyStats`          | Agent + Controller | `true`  | Beta  | v0.10         | v1.2         | N/A        | No                  |                                               |
+| `NodePortLocal`               | Agent              | `true`  | Beta  | v0.13         | v1.4         | N/A        | Yes                 | Important user-facing change in v1.2.0        |
+| `Egress`                      | Agent + Controller | `true`  | Beta  | v1.0          | v1.6         | N/A        | Yes                 |                                               |
+| `NodeIPAM`                    | Controller         | `true`  | Beta  | v1.4          | v1.12        | N/A        | Yes                 |                                               |
+| `AntreaIPAM`                  | Agent + Controller | `false` | Alpha | v1.4          | N/A          | N/A        | Yes                 |                                               |
+| `Multicast`                   | Agent + Controller | `true`  | Beta  | v1.5          | v1.12        | N/A        | Yes                 |                                               |
+| `SecondaryNetwork`            | Agent              | `false` | Alpha | v1.5          | N/A          | N/A        | Yes                 |                                               |
+| `ServiceExternalIP`           | Agent + Controller | `false` | Alpha | v1.5          | N/A          | N/A        | Yes                 |                                               |
+| `TrafficControl`              | Agent              | `false` | Alpha | v1.7          | N/A          | N/A        | No                  |                                               |
+| `Multicluster`                | Agent + Controller | `false` | Alpha | v1.7          | N/A          | N/A        | Yes                 | Controller side feature gate added in v1.10.0 |
+| `IPsecCertAuth`               | Agent + Controller | `false` | Alpha | v1.7          | N/A          | N/A        | No                  |                                               |
+| `ExternalNode`                | Agent              | `false` | Alpha | v1.8          | N/A          | N/A        | Yes                 |                                               |
+| `SupportBundleCollection`     | Agent + Controller | `false` | Alpha | v1.10         | N/A          | N/A        | Yes                 |                                               |
+| `L7NetworkPolicy`             | Agent + Controller | `false` | Alpha | v1.10         | N/A          | N/A        | Yes                 |                                               |
+| `AdminNetworkPolicy`          | Controller         | `false` | Alpha | v1.13         | N/A          | N/A        | Yes                 |                                               |
```

Comment:
Why is this table change? you may add a new row or change the necessary pieces instead of reformat it, it will be hard to review what's changed.

Reply:
I adjust the blank, now it only displays the changed line.